__tests__/app.test.js

```diff
@@ -13,3 +21,24 @@ describe("GET /api", () => {
       });
   });
 });
+
+describe("GET /api/topics", () => {
+  test("200: Responds with all topics", () => {
+    return request(app)
+    .get("/api/topics")
+    .expect(200)
+    .then(({body})=>{
+      expect(body.topics.length).toBe(3);
+
+      body.topics.forEach((topic) => {
+        expect(typeof topic.description).toBe("string")
+        expect(typeof topic.slug).toBe("string")
+      })
+    })
+  });
```

Comment:
Not a bad test, you can also structure it like this....

expect(topic).toMatchObject({
                        description: expect.any(String),
                        slug: expect.any(String)
                    })